src/pattern-library/components/patterns/prototype/TrabsitionComponentExperimentsPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { useState } from 'preact/hooks';
+
+import { Dialog } from '../../../../components/feedback';
+import { Button } from '../../../../components/input';
+import { Slider } from '../../../../components/transition';
```

Comment:
Minor nit, but I typically import from the package entry point in pattern library page component modules.